packages/world/mud.config.ts

```diff
@@ -87,6 +87,7 @@ export default defineWorld({
       schema: {
         recipeId: "bytes32",
         stationTypeId: "ObjectType",
+        smeltTime: "uint128",
```

Comment:
We'll need to update the client recipe ids to include smeltTime.

The furnace UI can happen after though and doesnt need to block this PR